packages/skill-api/src/core/services/stripe-checkout.ts

```diff
@@ -346,17 +387,6 @@ export async function stripeCheckout({
         throw new CheckoutError('No product was found', productId as string)
       }
 
-      const price =
-        loadedProduct.merchantProducts?.[0].merchantPrices?.[0].identifier
-
-      if (!price) {
-        throw new CheckoutError(
-          'no-pricing-available',
-          loadedProduct.id,
-          couponId as string,
-        )
-      }
```

Comment:
We extract this value and throw `CheckoutError` as necessarily at the top of the function, no need to do it here